website/src/components/HomepageFeatures/index.js

```diff
@@ -25,40 +93,111 @@ const FeatureList = [
   },
   {
     title: 'Accurate',
```

Comment:
Let's limit this to one section / 3 blurbs. Can we do:

- Population-scale capabilities: used for SV discovery in flagship research studies including the Genome Aggregation Project (gnomAD) and All of Us.
- Sensitive and accurate: ensemble calling with multiple SV discovery tools combined with joint genotyping maximize power, and ML-based variant adjudication filters poor quality variants.
- Cloud-native: built for the Terra genomics platform, enabling scalability, collaboration, and reproducibility in a secure environment.

Reply:
I updated the text; please check. Or, you prefer one paragraph with the content you shared? We can change it to a single paragraph, but it could read like a wall of text :)

Reply:
I also pushed a commit formatting the section as a single paragraph. Please check both commits and we can choose the one reads best.

Reply:
Thanks this is fine if it doesn't ft with the previous format. Can we also move this directly above the Projects section and remove the following Evidence Collection and QC, etc. row?

Reply:
> Thanks this is fine if it doesn't ft with the previous format.
It fits in the previous format (i.e., three columns), I can revert to that commit if you prefer that. 

 > Can we also move this directly above the Projects section and remove the following Evidence Collection and QC, etc. row?

Sure! Done!